files/docker.sh

```diff
@@ -11,15 +11,17 @@ if [[ -n $(sed -n -e "s/^DOCKER_APP_NAME=//p" .env ) ]]; then
   DOCKER_APP_NAME=$(sed -n -e "s/^DOCKER_APP_NAME=//p" .env )
 fi
 
-docker-compose exec mysql mysql -h127.0.0.1 -uroot -proot \
+sleep 30
```

Comment:
Why sleep against checker for DB available?

Reply:
Because ansible + check in the script (#temporary fix for mysql) enters a constant cycle in which the base which is even available is not checked, but simply shows that - We have been waiting for MySQL too long already; failing

Reply:
Using sleep not a good idea